modules/libarchive/3.7.5.bcr.1/presubmit.yml

```diff
@@ -0,0 +1,26 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform:
+      - debian10
+      - debian11
+      - ubuntu2004
+      - ubuntu2204
+      - macos
+      - macos_arm64
+      - windows
+    bazel: [6.x, 7.x, rolling]
```

Comment:
i think the presubmit is failing due to the bazel compability change. Removing `6.x` and adjusting the 7 to be `>=7.2.1` (assuming the presubmit syntax is the same)

Reply:
done